docker-compose.yaml

```diff
@@ -0,0 +1,52 @@
+
+version: '3'
+services:
+    sui-server:
+        image: sui-dev-image
+        init: true
+        ports:
+            - "4100:4100"
+            - "4200:4200"
+        volumes:
+            # base_app
+            - type: bind
+              source: "${PWD}/base_app"
+              target: /usr/src/app/base_app/
+            # sui-server
+            - type: bind
+              source: "${PWD}/simpleui-server"
+              target: /usr/src/app/simpleui-server/
+            # install-script.bash
+            - type: bind
+              source: "${PWD}/develop_docker/install-script.bash"
+              target: /usr/src/app/install-script.bash
+            # install-script-debug-simpleui-server.bash
+            - type: bind
+              source: "${PWD}/develop_docker/install-script-debug-simpleui-server.bash"
+              target: /usr/src/app/install-script-debug-simpleui-server.bash
+            # sample data
+            - type: bind
+              source: "${SIMPLEUI_TEST_DATA}/sample-app/opt"
+              target: /opt
+            - type: bind
+              source: "${SIMPLEUI_TEST_DATA}/sample-app"
+              target: /var/www/simple_ui
+        #entrypoint: ["/bin/bash", "/usr/src/app/install-script-debug-simpleui-server.bash"]       
+        entrypoint: ["/bin/bash", "/usr/src/app/install-script.bash"]
+
+    zmq_backend:
+        image: zmq_backend
+        hostname: svcmachineapps
+        init: true
+        environment:
+          PYTHONUNBUFFERED: 1
+        volumes:
+            # python file
+            - type: bind
+              source: "${PWD}/docker_zmq_backend/backend.py"
+              target: /app/backend.py
+            # sample data
+            - type: bind
+              source: "${SIMPLEUI_TEST_DATA}/sample-app/"
+              target: /app/data/
+        entrypoint: ["python", "backend.py"]
```

Comment:
Might be worth moving this file to docker_zmq_backend or maybe develop_docker. Basically just want to de-clutter the top level of the repo wherever possible. 

Also, should change your IDE settings to make sure files have exactly one newline at the end of the file

Reply:
moved docker-compose to `develop_docker`. Also updated my vscode settings